openomni/bin/omni_listen_rfcat

```diff
@@ -45,7 +45,8 @@ def main(options=None):
     with open('../../results/' + fileName + '.json', 'a') as outFile:
         """Create 1 dict for each message loop."""
         package ={}
-        messages = []
+        messages = {"string": [], 
```

Comment:
I've grouped the raw's all together and the strings, I first had it grouped for each step, but that was not really easy to read, see added example with the grouping I find is really helpful. @ps2 what do you think?